include/links-abstract-domain.php

```diff
@@ -67,8 +68,15 @@ protected function set_home_url( $language ) {
 	 * @return string The modified url.
 	 */
 	public function site_url( $url ) {
+		if ( ! $this->model->are_languages_created() ) {
```

Comment:
In order to simplify, could we use `PLL_Model::has_languages()` instead of introducing this new method?

Reply:
It appears that `has_languages()` doesn't do the job. Indeed it check also for languages term in the database. Whereas `are_languages_created()` tells us when language objects are created and that's what we want to know before filtering URLs.
I propose a better naming in ff7c22e.